CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ find_package(Boost 1.54.0 COMPONENTS system regex thread REQUIRED)
 include_directories(${Boost_INCLUDE_DIR})
 
 if(APPLE)
-  set(OPENSSL_ROOT_DIR "/usr/local/opt/openssl")
```

Comment:
Line 22 should be correct when using homebrew. But double check by writing `brew info openssl`. You should get something like:
```sh
LDFLAGS:  -L/usr/local/opt/openssl/lib
CPPFLAGS: -I/usr/local/opt/openssl/include
```